AutoRest/Generators/Ruby/Ruby/Templates/MethodTemplate.cshtml

```diff
@@ -95,11 +95,7 @@ def @(Model.Name)_async(@(Model.MethodParameterDeclaration))
       if (parameter.IsConstant)
       {
   @:@(parameter.Name) = @(parameter.DefaultValue)
-      }
-      else
-      {
-  @:@(parameter.Type.ValidateType(Model.Scope, parameter.Name))

```

Comment:
This has been moved into the serialization logic @ [value.validate](https://github.com/Azure/autorest/pull/1106/files#diff-439d867040fd69d4c723f03d46ae68d8R262)